CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # jsonld-signatures ChangeLog
 
 ### Changed
-- Updated jsonld to 1.5.0.
+- **NOTE**: Updated jsonld to 1.5.0. May require code updates to continue using
```

Comment:
I would say `Code updates will be required if...`

And perhaps a link to specific documentation if available.